ui/README.md

```diff
@@ -0,0 +1,47 @@
+
+
+[![License](http://img.shields.io/:license-apache%202.0-brightgreen.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
```

Comment:
Same above; could you incorporate the the UI build steps into the root README.

Reply:
sure